_layouts/pay-stamp-duty-land-tax.html

```diff
@@ -7,6 +7,42 @@
+<!--<![endif]-->
+    <!--[if lt IE 9]><script src="https://assets.publishing.service.gov.uk/static/ie-75db9d4df4993de5335d6e49013362743a78107245bcf6543337a0ecc0a95b41.js"></script><![endif]-->
+
+    <link rel="shortcut icon" href="../assets/favicon-8d811b8c3badbc0b0e2f6e25d3660a96cc0cca7993e6f32e98785f205fc40907.ico" type="image/x-icon">
+    <link rel="mask-icon" href="https://assets.publishing.service.gov.uk/static/gov.uk_logotype_crown-a0097118fe9226ce1ffde1ccfeb86c61be3607915046196c99f68b9548485ad3.svg" color="#0b0c0c">
+    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="https://assets.publishing.service.gov.uk/static/apple-touch-icon-152x152-55dd2af7a53abb7dd14ba6a3c6c3fc786c57a074ba809f6dd767e186955a87e1.png">
+    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="https://assets.publishing.service.gov.uk/static/apple-touch-icon-120x120-92580784afc35143f96b8cf5614e221427fdea444aeb13d912c879e884af7336.png">
+    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="https://assets.publishing.service.gov.uk/static/apple-touch-icon-76x76-9b193f39cde53b92ba555efed856bcd103d6d1dc650919eb9f29dbe5f442cfb0.png">
+    <link rel="apple-touch-icon-precomposed" href="https://assets.publishing.service.gov.uk/static/apple-touch-icon-60x60-0af0a7cf886701647210f13b661e607b7223bc3221b6d275718a89af79b3e7b9.png">
+
+    <meta name="theme-color" content="#0b0c0c">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:image" content="https://assets.publishing.service.gov.uk/static/opengraph-image-a1f7d89ffd0782738b1aeb0da37842d8bd0addbd724b8e58c3edbc7287cc11de.png">
+
+      <!--[if gt IE 8]><!--><link rel="stylesheet" media="screen" href="https://assets.publishing.service.gov.uk/static/core-layout-f1cff307a88ad244e2ea5c53b3650dc6b3a4b8cf34cbde84717d1d20e95ca931.css">
+<!--<![endif]-->
+<!--[if IE 6]><link rel="stylesheet" media="screen" href="https://assets.publishing.service.gov.uk/static/core-layout-ie6-510bb7d04749a5e1598b2acaa2e0e78e2aa2a26a36ec4472578b9f486f47cdc2.css" /><script>var ieVersion = 6;</script><![endif]-->
+<!--[if IE 7]><link rel="stylesheet" media="screen" href="https://assets.publishing.service.gov.uk/static/core-layout-ie7-dd5202407e2eb55f275d31c6a5df5bbf44b2b75e163ca6ec309f042b399eabb6.css" /><script>var ieVersion = 7;</script><![endif]-->
+<!--[if IE 8]><link rel="stylesheet" media="screen" href="https://assets.publishing.service.gov.uk/static/core-layout-ie8-e696c814adbd6fb6a7e0ae454b5412c0e566f286e23f27bb88ad77eb4493c9d2.css" /><script>var ieVersion = 8;</script><![endif]-->
+
+<link rel="stylesheet" media="print" href="../assets/core-layout-print-522a1388d9519dbd9796113904b7383d3ad6d7939a5f4c9318282d7a3e92f0c1.css">
+
+
+  <!--[if gt IE 8]><!--><link rel="stylesheet" media="screen" href="../assets/application-136d1402a3ff580fbe4df2ba095ea2d0daa33ed65e60f37299a441d73baa3aab.css">
```

Comment:
You shouldn't need all of these bits as their covered in the header-assets include. If you could put your own css file in there that'd be great.